Docs Homepage (#1082)
nuxt/content: docs/content/1.index.md

  - https://github.com/nuxt/content
snippet: yarn add --dev @nuxt/content
---
#title
Content made easy  
for developers
#description
Write pages in markdown, use Vue components and enjoy  
the power of Nuxt with a blazing fast developer experience.
::

::n-section
  ::n-hero
  ---
  tag: Features
  icon: heroicons-solid:sparkles
  ---
  #title
  Store your Content at  
  the right place.
  #description

>   Static content should be store in files instead of databases. No need to learn a new query language or complicated API docs to read. Fetch your content as data and display it directly in your components.

Reviewer: ```suggestion
  Static content should be stored in files instead of databases. No need to learn a new query language or read complicated API docs. Fetch your content as data and display it directly in your components.
```

  ::
  ::n-grid
    ::n-feature
    ---
    icon: uil:map-marker-edit
    ---
    #title
    Files-based CMS
    
    #description

>     Write content in mardkdown, yml, csv or json and query it in your components.

Reviewer: ```suggestion
    Write content in markdown, YAML, CSV or JSON and query it in your components.
```

  - https://github.com/nuxt/content
snippet: yarn add --dev @nuxt/content
---
#title
Content made easy  
for developers
#description
Write pages in markdown, use Vue components and enjoy  
the power of Nuxt with a blazing fast developer experience.
::

::n-section
  ::n-hero
  ---
  tag: Features
  icon: heroicons-solid:sparkles
  ---
  #title
  Store your Content at  
  the right place.
  #description
  Static content should be store in files instead of databases. No need to learn a new query language or complicated API docs to read. Fetch your content as data and display it directly in your components.
  ::
  ::n-grid
    ::n-feature
    ---
    icon: uil:map-marker-edit
    ---
    #title
    Files-based CMS
    
    #description
    Write content in mardkdown, yml, csv or json and query it in your components.
    ::
    ::n-feature
    ---
    icon: uil:brackets-curly
    ---
    #title
    Query Builder
    
    #description
    Query your content with a MongoDB-like API to fetch the right data at the right time.
    ::
    ::n-feature
    ---
    icon: uil:document-layout-center
    ---
    #title

>     MDC Syntax

Reviewer: ```suggestion
    MDX Syntax
```

    
    #description
    Use Vue components in .md files, supporting props, slots and nested components.
    ::
    ::n-feature
    ---
    icon: uil:bolt-alt
    ---
    #title

>     Production Ready

Reviewer: ```suggestion
    Production-ready
```

    
    #description

>     Take advantage of Nuxt 3 and its features such as Vue 3, Auto-imports, Vite and Nitro server.

Reviewer: ```suggestion
    Take advantage of Nuxt 3 and its features such as Vue 3, auto-imports, Vite and Nitro server.
```
